Mathlib/Algebra/Module/Zlattice/Basic.lean

```diff
@@ -522,7 +522,7 @@ theorem Zlattice.rank [hs : IsZlattice K L] : finrank ℤ L = finrank K E := by
     exact Subtype.coe_injective.comp (Basis.injective _)
   rw [finrank_eq_card_chooseBasisIndex]
     -- We prove that `finrank ℤ L ≤ finrank K E` and `finrank K E ≤ finrank ℤ L`
-  refine le_antisymm ?_ ?_
+  apply le_antisymm
```

Comment:
I don't think this should be changed. A `refine` call with two or more `?_` holes tells the readers how many new goals will be created. If I see `apply`, I would only expect one goal to be left afterwards.

Reply:
There are 2745 matches for `apply.*\n *·`, so it seems in those cases there is more than one goal left.

Reply:
The reader can also see how many new goals are created by looking at the well structured indented "`·`".